code/_onclick/hud/screen_objects.dm

```diff
@@ -671,7 +671,7 @@
 /atom/movable/screen/splash
 	icon = 'icons/blanks/blank_title.png'
 	icon_state = ""
-	screen_loc = "1,1"
+	screen_loc = "CENTER-7,CENTER-7" // Why here? The old is 1,1 - which makes it at the bottom left corner. Jank! This will avoid alignment issues altogether.
```

Comment:
what does this change?

Reply:
It changes the splash screen from like a bottom-left-align format to an absolute center format. The images are 480x480, and center-7,center-7 is equal to 1,1's coordinates on 15x15, but if you go over 15x15 the image will still always be on the bottom corner before.